src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1510,10 +1511,36 @@ public void deleteOrganizationMembership(long user_id, long organization_members
     }
 
     public <T extends SearchResultEntity> Iterable<T> getSearchResults(Class<T> type, String query) {
-        return getSearchResults(type, query, null);
+        return getSearchResults(type, query, Collections.emptyMap());
     }
 
+    /**
+     * Use {@link #getSearchResults(Class, String, Map)} instead.
```

Comment:
could be 

```
@deprecated Use {@link ...
```

Reply:
Thinking about it we could keep both methods with deprecating one. This one, without a `SortOrder` could offer a default order.

Reply:
+1 I think it is what @PierreBtz did for Comment ordering